src/Juvix/Compiler/Backend/Isabelle/Language.hs

```diff
@@ -154,13 +154,15 @@ data Statement
 data Definition = Definition
   { _definitionName :: Name,
     _definitionType :: Type,
-    _definitionBody :: Expression
+    _definitionBody :: Expression,
+    _definitionComment :: Maybe Text
```

Comment:
Perhaps it would be more appropriate we named this `Doc` or `DocComment` rather than `Comment`